docs/technical_reference/flowsheets/electrodialysis_1stack_conc_recirc.rst

```diff
@@ -0,0 +1,116 @@
+Documentation for the property model:
+    * `Multi-Component Aqueous Solution (MCAS) Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/mc_aq_sol.html>`_
+
+Degrees of Freedom
+------------------
+The number of degrees of freedom (DOF) is associated with the number of fixed variables (parameters) determined by the purpose of the modeling case. We implemented two modeling cases in the flowsheet: (1) the prediction of desalination outcome (salinity of the product water and saline disposal) and (2) the optimization of key decision variables in system design.  In the first case, DOF is set to zero by fixing all initial conditions of the feed solution fluid and definite ED stack parameters. All fixed values are presented in the section to follow.  In the second case, the values of those chosen to be the decision variables in the optimization are unfixed. The DOF number is therefore the number of decision variables. In this example, the decision variables are
+
+    * stack voltage applied
+    * ED cell pair number 
+    * ED cell length
+
+Flowsheet Specifications
+------------------------
+.. csv-table:: 
+   :header: Name, Value, Unit, Reference
+   :widths: 30, 20, 20, 10
+
+   "Salinity (NaCl)", ":math:`2`", ":math:`g L^{-1}`", "--"
+   "Volume flow rate", ":math:`5.2 \times 10^{-4}`", ":math:`m^3 s^{-1}`", [2]_
+   "Temperature", ":math:`298.15`", ":math:`K`", "--"
+   "Pressure", ":math:`101325`", ":math:`Pa`", "--"
+   "Na^+ diffusivity", ":math:`1.33 \times 10^{-9}`", ":math:`m^2 s^{-1}`",[3]_
+   "Cl^- diffusivity", ":math:`2.03 \times 10^{-9}`", ":math:`m^2 s^{-1}`",[3]_
+   "NaCl mass diffusivity", ":math:`1.60 \times 10^{-9}`", ":math:`m^2 s^{-1}`", [4]_
+   "Cell pair number", ":math:`56`", ":math:`1`", "--"
+   "Cell length", ":math:`1.68`", ":math:`m`", [2]_
+   "Cell width", ":math:`0.197`", ":math:`m`",[5]_
+   "Channel height", ":math:`7.1 \times 10^{-4}`", ":math:`m`", "--"
+   "Water recovery", ":math:`70%`", ":math:`1`", "--"
+   "Stack voltage", ":math:`10`", ":math:`V`", "--"
+   "Thickness, aem and cem", ":math:`1.3 \times 10^{-5}`", ":math:`m`",[5]_
+   "Areal resistance, aem", ":math:`1.77 \times 10^{-4}`", ":math:`\Omega m^2`", [5]_
+   "Areal resistance, cem", ":math:`1.89 \times 10^{-4}`", ":math:`\Omega m^2`",[5]_
+   "Water permeability, aem", ":math:`1.75 \times 10^{-14}`", ":math:`m s^{-1} Pa^{-1}`",[5]_
+   "Water permeability, cem", ":math:`2.16 \times 10^{-14}`", ":math:`m s^{-1} Pa^{-1}`", [5]_
+   "Water transport number, aem", ":math:`4.3`", ":math:`1`",[6]_
+   "Water transport number, cem", ":math:`5.8`", ":math:`1`", [7]_
+   "NaCl mass diffusivity, aem and cem", ":math:`3.28 \times 10^{-11}`", ":math:`m^2 s^{-1}`", [8]_
+   "Spacer porosity", ":math:`0.83`", ":math:`1`", [2]_
+   "Spacer specific surface area", ":math:`10400`", ":math:`m^{-1}`", [9]_
+   "Pump efficiency", ":math:`0.8`", ":math:`1`", "--"
+
+
+References
```

Comment:
Something is going on with these in the build that I'm not sure of
![image](https://github.com/watertap-org/watertap/assets/101219154/bc35e207-2f1e-4bdd-ba81-d42b2b0ec5f0)

Reply:
these are proper printout by the used referencing codes.  The numbers in "()" tell the subindices of places where that citation item is relevant.